doc/nrf/known_issues.rst

```diff
@@ -597,6 +597,14 @@ nRF Desktop
 
 .. rst-class:: v1-7-1 v1-7-0
 
+NCSDK-12337: Possible assertion fail while a USB connected host boots
+  While the USB connected host boots, the HID report subscriptions may be disabled and enabled a few times without disconnecting the USB.
+  That may result in improper subscription handling and assertion failure in :ref:`nrf_desktop_hid_state`.
+
+  **Workaround:** Manually cherry-pick and apply commit with fix from main.
```

Comment:
Which commit should be cherry-picked? Can you provide the SHA here?

Reply:
I will merge commit with fix first and then we will add a known issue (currently I cannot refer to commit hash, because fix is not merged yet).

Reply:
Added